packages/core/src/index.ts

```diff
@@ -101,7 +130,7 @@ class ImageService extends Service {
     return this.ctx
       .http(url, { method: 'GET', responseType: 'arraybuffer', proxyAgent: '' })
       .then((resp) => {
-        return `data:${resp.headers['content-type']};base64,${Buffer.from(resp.data).toString('base64')}`
+        return `data:${resp.headers.get('content-type')};base64,${Buffer.from(resp.data).toString('base64')}`
```

Comment:
This change can be split in a separated fixup PR.

Other changes are still in consideration.